### PR TITLE
Eg/refactor end checkout

### PR DESCRIPTION
**Pull request by eigiachero**
Needs https://github.com/WeDevelop-ARG/cg-frontend/pull/159.

### CODE REVIEW

File: src/modules/Store/pages/Checkout/PurchaseSuccess/ProductShare/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import { FaceBook, Instagram, Twitter } from '~/src/components/SocialMediaIcons'
+
+import CopyLink from '~/src/vectors/copy-link.svg'
+import classes from './styles.module.scss'
+
+const ProductShare = ({ productName }) => {
+  return (
+    <div className={classes.share}>
+      <div className={classes.shareBox}>
+        <p className={classes.productUrl}>http://comprasgrupales.com/{(productName).replace(/ /g, '-').toLowerCase()}</p>
```
Comment (nahudalla) [inline src/modules/Store/pages/Checkout/PurchaseSuccess/ProductShare/index.js]: :thinking: I'd maybe use an environment variable to configure the website's base URL. Also, **always** use HTTPS.

**Review by nahudalla — CHANGES_REQUESTED** [review]

File: src/modules/Store/pages/Checkout/PurchaseSuccess/ProductShare/index.js (hunk 1 above)
Comment (nahudalla) [reply src/modules/Store/pages/Checkout/PurchaseSuccess/ProductShare/index.js]: You can configure environment variables like this one in the `cloudbuild_xxx.yaml` files in the root of the repo.

**Review by nahudalla — APPROVED** [review]

File: src/modules/Store/pages/Checkout/PurchaseSuccess/ProductInfo/index.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import ImageAsBackround from '~/src/modules/MainApp/components/ImageAsBackground'
+
+import PriceTag from '~/src/vectors/price-tag.svg'
+import classes from './styles.module.scss'
+
+const EndCheckout = ({ product, discount }) => {
+  return (
+    <div className={classes.product}>
+      <div className={classes.productData}>
+        <ImageAsBackround imageUrl={product.photos[0].url} className={classes.image} />
```
Comment (MPucheta) [inline src/modules/Store/pages/Checkout/PurchaseSuccess/ProductInfo/index.js]: Typo `backround`

File: src/modules/Store/pages/Checkout/PurchaseSuccess/ProductInfo/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,104 @@
+@import '~src/styles/mixin/media-screen.scss';
+
+.product {
+  display: flex;
+  width: 100%;
+  border-top: 1px solid #F1F3F3;
```
Comment (MPucheta) [inline src/modules/Store/pages/Checkout/PurchaseSuccess/ProductInfo/styles.module.scss]: Put color variables

File: src/modules/Store/pages/Checkout/PurchaseSuccess/ProductShare/styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  width: 100%;
+}
+
+.shareBox {
+  width: 540px;
+  display: flex;
+  padding: 8px 0;
+  padding-left: 12px;
+
+  background: #FFFFFF;
```
Comment (MPucheta) [inline src/modules/Store/pages/Checkout/PurchaseSuccess/ProductShare/styles.module.scss]: Use colors

File: src/modules/Store/pages/Checkout/PurchaseSuccess/index.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    history.push('/store')
+  }
+
+  const leftParticipants = group.minParticipants - (group.participantsCount + 1)
+  const discount = getDiscount(product.marketPrice, product.price)
+
+  return (
+    <>
+      <button className={classes.redirect} onClick={() => goToHome()}>
+        &#60;  Volver a la pagina principal
```
Comment (MPucheta) [inline src/modules/Store/pages/Checkout/PurchaseSuccess/index.js]: typo `página`, unnecesssary lambda `()=> goToHome()`

File: src/modules/Store/pages/Checkout/PurchaseSuccess/index.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { useHistory } from 'react-router-dom'
+import GroupProgress from '~/src/components/GroupProgress'
```
Comment (MPucheta) [inline src/modules/Store/pages/Checkout/PurchaseSuccess/index.js]: Replace with the GroupProgress inside `MainApp`

File: src/modules/Store/pages/Checkout/PurchaseSuccess/index.js [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+        <span>Te enviaremos un mail de confirmación cuando el grupo se haya completado y la compra se haga efectiva.</span>
+        <ProductInfo product={product} discount={discount} />
+        <GroupProgress
+          currentParticipants={group.participantsCount}
+          minParticipants={group.minParticipants}
+          type={group.type}
+        />
+        <div className={classes.leftParticipants}>
+          <p><b>Faltan {leftParticipants} compradores para completar el grupo!</b></p>
+          <p>Compartí y suma gente para acceder más rápido a tu producto</p>
```
Comment (MPucheta) [inline src/modules/Store/pages/Checkout/PurchaseSuccess/index.js]: `sumá`

File: src/modules/Store/pages/Checkout/PurchaseSuccess/styles.module.scss [hunk 8]
```diff
@@ -0,0 +1,76 @@
+@import '~src/styles/mixin/media-screen.scss';
+
+.container {
+  max-width: 1200px;
+  margin: 100px auto 0;
+  padding: 65px 110px 70px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  background: #FFFFFF;
```
Comment (MPucheta) [inline src/modules/Store/pages/Checkout/PurchaseSuccess/styles.module.scss]: Use color variables

**Review by MPucheta — COMMENTED** [review]
This is a "self-review", I will change them right away

**Pull request merged**